customize.dist/translations/messages.es.js

```diff
@@ -18,7 +18,7 @@ define(function () {
     out.synchronizing = "Sincronización";
     out.reconnecting = "Reconectando...";
     out.lag = "Retraso";
-    out.readonly = 'Solo lectura';
+    out.readonly = 'Sólo lectura';
```

Comment:
I believe this is incorrect: http://www.rae.es/consultas/el-adverbio-solo-y-los-pronombres-demostrativos-sin-tilde